recipes/puremagic/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2013-2017 Chris Griffith
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
```

Comment:
This file shouldn't needed as, according to this line, there should be a copy of the licence included with every distribution of the package. It is included in the GitHub tarball but isn't in the PyPI release. It might be worth making an issue or PR upstream to fix this.

It should be automatically included in the release tarballs on PyPI if you edit the `setup.cfg` to include:

```
[metadata]
license_files = LICENSE
```

See [here (and the sample project)](https://packaging.python.org/guides/distributing-packages-using-setuptools/#license-txt) for more details.

Reply:
If you continue using the tarball from GitHub this file can be deleted, else if you use the PyPI tarball it will need to be kept for now.